builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -13,6 +13,22 @@ temp_app_source_dir="/tmp/appsource"
 temp_app_source_path="$temp_app_source_dir/$FILE_UPLOAD_BLOB_NAME"
 mkdir $temp_app_source_dir
 
+
+# List all the environment variables and filter the environment variable with prefix "USER_ENV_", then write them to folder "/platform/env", 
+# file name is the environment variable name without prefix USER_ENV_, file content is the environment variable value.
+build_env_dir="/platform/env"
```

Comment:
nit: do we need to create this `/platform/env` folder before we start writing files to it?

Reply:
/platform/env folder already exists in the builder image, no need to create it.